testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/ClientInitiatedAccountLinkTest.java

```diff
@@ -560,12 +555,11 @@ public void testAccountLinkingExpired() throws Exception {
         Assert.assertTrue(links.isEmpty());
 
         // Login to account mgmt first
-        profilePage.open(CHILD_IDP);
+        driver.navigate().to(RealmsResource.accountUrl(UriBuilder.fromUri(getAuthServerRoot())).build(CHILD_IDP).toString());
```

Comment:
We shouldn't navigate to the account url here, as that will open the new admin console. I'd change this to use the test app instead.